RecoLocalCalo/HcalRecAlgos/interface/DoMahiAlgo.h

```diff
@@ -0,0 +1,146 @@
+#ifndef RecoLocalCalo_HcalRecAlgos_DoMahiAlgo_HH
+#define RecoLocalCalo_HcalRecAlgos_DoMahiAlgo_HH
+
+#include "DataFormats/HcalDetId/interface/HcalSubdetector.h"
+#include "DataFormats/HcalDetId/interface/HcalDetId.h"
+#include "CalibFormats/HcalObjects/interface/HcalCalibrations.h"
+#include "CalibFormats/HcalObjects/interface/HcalCoder.h"
+#include "RecoLocalCalo/HcalRecAlgos/interface/EigenMatrixTypes.h"
+#include "DataFormats/HcalRecHit/interface/HBHEChannelInfo.h"
+
+#include "CalibCalorimetry/HcalAlgos/interface/HcalPulseShapes.h"
+#include "CalibCalorimetry/HcalAlgos/interface/HcalTimeSlew.h"
+#include "RecoLocalCalo/HcalRecAlgos/interface/PulseShapeFunctor.h"
+
+#include "Math/Functor.h"
+
+class DoMahiAlgo
+{
+ public:
+  DoMahiAlgo();
+  ~DoMahiAlgo() { };
+
+  void phase1Apply(const HBHEChannelInfo& channelData, float& reconstructedEnergy, float& reconstructedTime, float& chi2);  
+  bool DoFit(SampleVector amplitudes, std::vector<float> &correctedOutput, int nbx);
```

Comment:
can you rename this and all the methods below to follow the CMSSW naming conventions, i.e., begin with a lowercase letter ?